### DISCUSSION TITLE
GPU Auto Stop/Start Function

### DISCUSSION THREAD

**Discussion by Y4m4k** [Rem0o/FanControl.Releases#607] [Q&A] [answered]
Hi - when I installed the Fan Control program, it seems to have disable the GPU auto start/stop function. My fans are continually spinning even though the GPU temp is 42C. Usually they would be stopped at this temp. I have the controls set off so I would assume that GPU drivers/firmware would take control. Any way to bring it back if I don't want to change the GPU fan curves? Thank you for putting this great program together!

Dan

![image](https://user-images.githubusercontent.com/86619370/164873521-b0aa4256-6db7-4e4c-89b5-db4c7d966c1e.png)

**Comment by xhul-dev** [comment]
Hello there.
That can happen with non-GPU fans aswell.
Try closing Fan Control completely ("exit" from system tray).
If your GPU fans continue to spin where they shouldn't, restart the system, that one's guarenteed to work.

**Reply by Y4m4k** [reply]
Hi - thanks for your thoughts. I tried that, didn't bring back the nvidia fan curve.

**Comment by xhul-dev** [comment]
When at low GPU temp, did you actually verify if the fans are still spinning _physically_?
Cause if you read the RPM with Fan Control after a restart, you might continue to read an incorrect value, especially if you checked the "Start with Windows" option.
To be sure, i suggest going into the BIOS when the GPU is cold enough, and check the fans with your own eyes.

Also, it could be that the fan pairing in Fan Control is incorrect, that would explain the wrong RPMs.
For example, even the RPM of your "Back Exhaust" looks suspicious in your screenshot (says 0% but 650 RPM).
I suggest you run the "Pair controls and speeds automatically..." function in Fan Control, maybe you'll get more accurate RPMs afterwards.

**Reply by Y4m4k** [reply]
Thanks Xhul - yes, I can physically see the GPU fans spinning (they are stuck at 55%). I checked the BIOS, there is not read out for GPU in there. You right, I couldn't get the back exhaust to stop too. 

I did a system restore, that didn't help. So I did a DDU on the GPU. It solved the issue. Not sure what Fan Control did to the drivers. How do I do a clean uninstall of Fan Control and start over? Or is it portable so there is no uninstall process?

Thanks for the help!

**Reply by Rem0o** [reply]
It's portable.

**Reply by Y4m4k** [reply]
### **So, it turns out it wasn't Fan Control related 😁!!**

I know what the issue was... it because of my setting in the Nvidia Control Panel. I had custom settings. When I DDU and reinstalled the drivers, it wiped my settings back to default, hence, GPU under less load, no fan spin. Thats when I messaged you. 

I went back to Nvidia Control Panel and re-did my settings, and the issue came back. Even though the temps were around 40-45C, the wattage was around 100-115W (at default Nvidia Control Panel Settings, the GPU wattage is currently at ~20W however at the same temp ~40-45C). I believe that is why the GPU fans were not shutting down.

Sorry about that... at least we know something new now!

![image](https://user-images.githubusercontent.com/86619370/164924856-e0fe062f-5ea4-42e4-acea-0fc82b2ed1ba.png)

**Comment by Y4m4k** [comment, marked as answer]
https://github.com/Rem0o/FanControl.Releases/discussions/607#discussioncomment-2621578

**Comment by Optimus-Bull** [comment]
Didn't help for me. I did what you said and my fans did went to 0 RPM all the way until I enabled the GPU Fans in Fan Control. Then it hops back to 30% when the temperatures tells it to, but never goes back to 0% when the temps have lowered to the point where the fans should stop. If I disable the GPU Fans in Fan Control then it will go back to 0 RPM without any issues.

I uninstalled MSI afterburner, started Windows 11 in safe mode, ran DDU to completely remove the Nvidia Driver, went back into Windows to re-install the drivers and haven't changed a thing and yet the GPU Fans stays at 30%.

To me, Fan Control behaves exactly like MSI Afterburner did where it wouldn't allow me to turn fans off until I disabled the fan curve settings.

**Reply by Y4m4k** [reply]
Have a look at this thread, I think it might solve your issue —> https://github.com/Rem0o/FanControl.Releases/discussions/687

**Reply by Rem0o** [reply]
The 30% thing is a nvidia driver hook FanControl doesn't have access too. Some newer cards require it to go 0 rpm.
